src/renderers/webgl/WebGLBindingStates.js

```diff
@@ -0,0 +1,573 @@
+﻿/**
+ * @author Mugen87 / https://github.com/Mugen87
+ * @author Takahiro / https://github.com/takahirox
+ */
+
+function WebGLBindingStates( gl, extensions, attributes, capabilities ) {
+
+	var maxVertexAttributes = gl.getParameter( gl.MAX_VERTEX_ATTRIBS );
```

Comment:
@takahirox Can you please remove all usage of `var` from the PR? The core uses now `let` and `const`.

Reply:
Yes, that's the plan.

Reply:
Done